test/parallel/test-fs-realpath.js

```diff
@@ -197,7 +197,7 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); }, /ELOOP/);
+  assert.throws(function() { fs.realpathSync(entry); });
```

Comment:
I just reverted all changes to test-fs-realpath from @trevnorris commit. It was specific for C realpath implementation.